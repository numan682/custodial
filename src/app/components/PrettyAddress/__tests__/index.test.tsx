import * as React from 'react'
import { render } from '@testing-library/react'

import { PrettyAddress } from '..'

describe('<PrettyAddress  />', () => {
  it('should match snapshot', () => {
    const page = render(<PrettyAddress address="oasis1qq2vzcvxn0js5unsch5me2xz4kr43vcasv0d5eq4" />)
    expect(page.container.firstChild).toMatchSnapshot()
  })

  it('should split the address', () => {
    const page = render(<PrettyAddress address="oasis1qq2vzcvxn0js5unsch5me2xz4kr43vcasv0d5eq4" />)
    expect(page.container.firstChild).toMatchSnapshot()
    expect(page.container.firstChild!.textContent).toMatch(
      /oasis1 qq2v zcvx n0js 5uns ch5m e2xz 4kr4 3vca sv0d 5eq4/,
    )
  })
})
