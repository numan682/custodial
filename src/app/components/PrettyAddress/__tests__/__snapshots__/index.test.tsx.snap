// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrettyAddress  /> should match snapshot 1`] = `
.c0 {
  font-size: 18px;
  line-height: 24px;
}

<span
  class="c0"
  style="font-family: Roboto mono; letter-spacing: 0;"
>
  <span
    class="notranslate"
    translate="no"
  >
    oasis1 qq2v zcvx n0js 5uns ch5m e2xz 4kr4 3vca sv0d 5eq4
  </span>
</span>
`;

exports[`<PrettyAddress  /> should split the address 1`] = `
.c0 {
  font-size: 18px;
  line-height: 24px;
}

<span
  class="c0"
  style="font-family: Roboto mono; letter-spacing: 0;"
>
  <span
    class="notranslate"
    translate="no"
  >
    oasis1 qq2v zcvx n0js 5uns ch5m e2xz 4kr4 3vca sv0d 5eq4
  </span>
</span>
`;
